Makefile

```diff
@@ -7,14 +7,16 @@ all: openwrt
 openwrt: build_openwrt
 	echo "OpenWrt Done!"
 
-clean: clean_openwrt clean_feeds clean_binaries
+clean: clean_openwrt clean_feeds clean_binaries clean_keys
 
 # Building OpenWRT
 $(DIR__CI)/patched:
 	git submodule init openwrt;git submodule update --remote; \
 	cd $(DIR__OPENWRT); \
 	./scripts/feeds update -a; \
 	./scripts/feeds install -a;
```

Comment:
Think we need to add \ after each command?

    ./scripts/feeds update -a; \
    ./scripts/feeds install -a; \
    /vault read -field=key secret/creator/packagesigning > key.pem; \
    /vault read -field=cert secret/creator/packagesigning > cert.pem;